src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+import Icon from "../Icon/Icon";
+import { Item, Wrapper, AbsoluteWrapper, Current } from "./Dropdown.styles";
+
+export type SelectOption = {
+  label: string;
+  value: string;
+};
+
+export type DropdownProps = {
+  value?: SelectOption;
```

Comment:
```suggestion
  value: SelectOption;
```

In the designs there's always a default state. So `value` should not be optional.